### ISSUE TITLE
An important thing, but is not resolved all along: how to get the gradients of network w.r.t weights

### ISSUE THREAD

**Issue by jf003320018** [fchollet/keras#4201]
I have checked the related issues, and find that the issues how to get the gradients of network w.r.t weights is not sovled all along. 
New issues are open recently, but no one answer. 
So could any one give an example to get gradients of network w.r.t weights or bias in any layer? 
I think this is an important thing.

**Comment by igormq** [comment]
You can use [K.gradients](https://github.com/fchollet/keras/blob/master/keras/backend/tensorflow_backend.py#L1061) to do that.

**Comment by jf003320018** [comment]
@igormq I  have tried the methods in https://github.com/fchollet/keras/issues/2226, but I fail to get the gradients.  When I use K.gradients, it gives the tensor and I don't know how to use the K.function get the values.

**Comment by jf003320018** [comment]
The problem is solve in #2226. I will close this issue.
